sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -672,11 +672,11 @@ public partial class DataFeedIngestionProgress
     }
     public partial class DataFeedIngestionSettings
     {
-        public DataFeedIngestionSettings() { }
+        public DataFeedIngestionSettings(System.DateTimeOffset ingestionStartTime) { }
```

Comment:
In some cases, we had these empty constructors to support the following Update scenario:
```cs
var dataFeed = new DataFeed()
{
  // Only set what we want to update. No other property.
  IngestionSettings = new DataFeedIngestionSettings() { IngestionRetryDelay = TimeSpan.FromSeconds(30) }
};
adminClient.UpdateDataFeed(<dataFeedId>, dataFeed);
```

Since we're not supporting this scenario anymore (we would have problems when trying to differentiate "null" from "undefined"), the current approach requires customers to do:
```cs
DataFeed dataFeed = adminClient.GetDataFeed(<dataFeedId>);
dataFeed.IngestionSettings.IngestionRetryDelay = TimeSpan.FromSeconds(30);
adminClient.UpdateDataFeed(dataFeed);
```

`DataFeedIngestionSettings.IngestionStartTime` is a required property. We were not asking for it in the constructor to support the first scenario above, which we don't support anymore.